components/applicationsets/ztp-clusters-appset.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   generators:
     - git:
         repoURL: https://github.com/redhat-partner-solutions/vse-carslab-hub.git
-        revision: main
+        revision: cars2
```

Comment:
Does this mean we are only doing GitOps ZTP in the cars2 lab on a permanent basis?  Take it for @hhamid93 he would need to fork this himself then since the appset isn't using main branch anymore?

Reply:
We still have main branch. This PR is only for `cars2` branch so it will allow us to make changes in this lab without breaking main.